src/components/Header/Header.tsx

```diff
@@ -11,16 +11,38 @@ import SingUpIcon from '@assets/icons/user_add.svg?react';
 
 import { PageName } from '@/interfaces/Pages';
 import type { Product } from '@/interfaces/Product';
+import type { Themes } from '@/interfaces/Themes';
 
 import styles from './header.module.css';
 
+const htmlElement = document.querySelector('html');
```

Comment:
for future
i such small project it okey, but form something more serious, DON NOT work with native DOM elements inside the react apps, always try to avoid working with DOM

Reply:
Ok, thank you